release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -0,0 +1,1627 @@
+=== Scalability and performance
+
+[id="ocp-release-notes-etcd-certificates_{context}"]
+=== Security
+
+[id="ocp-release-notes-storage_{context}"]
+=== Storage
+
+[id="ocp-release-notes-web-console_{context}"]
+=== Web console
+
+[id="ocp-administrator-perspective_{context}"]
+==== Administrator perspective
+
+[id="ocp-developer-perspective_{context}"]
+==== Developer Perspective
+
+This release introduces the following updates to the *Developer* perspective of the web console:
+
+[id="ocp-4-19-notable-technical-changes_{context}"]
+== Notable technical changes
+
+[id="ocp-4-19-deprecated-removed-features_{context}"]
+== Deprecated and removed features
+
+Some features available in previous releases have been deprecated or removed.
+
+Deprecated functionality is still included in {product-title} and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments. For the most recent list of major functionality deprecated and removed within {product-title} {product-version}, refer to the table below. Additional details for more functionality that has been deprecated and removed are listed after the table.
+
+In the following tables, features are marked with the following statuses:
+
+* _Not Available_
+* _Technology Preview_
+* _General Availability_
+* _Deprecated_
+* _Removed_
+
+[discrete]
+[id="ocp-release-note-bare-metal-dep-rem_{context}"]
+=== Bare metal monitoring deprecated and removed features
+
+.Bare Metal Event Relay Operator tracker
+[cols="4,1,1,1",options="header"]
+|====
+|Feature |4.17 |4.18 |4.19
+
+|Bare Metal Event Relay Operator
+|Removed
+|Removed
+|Removed
+|====
+
+[discrete]
+[id="ocp-release-note-images-dep-rem_{context}"]
+=== Images deprecated and removed features
+
+.Images deprecated and removed tracker
+[cols="4,1,1,1",options="header"]
+|====
+|Feature |4.17 |4.18 |4.19
+
+|Cluster Samples Operator
+|Deprecated
+|Deprecated
+|Deprecated
+|====
+
+[discrete]
+[id="ocp-release-note-install-dep-rem_{context}"]
+=== Installation deprecated and removed features
+
+.Installation deprecated and removed tracker
+[cols="4,1,1,1",options="header"]
+|====
+|Feature |4.17 |4.18 |4.19
+
+|`--cloud` parameter for `oc adm release extract`
+|Deprecated
+|Deprecated
+|Deprecated
+
+|CoreDNS wildcard queries for the `cluster.local` domain
+|Deprecated
+|Deprecated
+|Deprecated
+
+|`compute.platform.openstack.rootVolume.type` for {rh-openstack}
+|Deprecated
+|Deprecated
+|Deprecated
+
+|`controlPlane.platform.openstack.rootVolume.type` for {rh-openstack}
+|Deprecated
+|Deprecated
+|Deprecated
+
+|`ingressVIP` and `apiVIP` settings in the `install-config.yaml` file for installer-provisioned infrastructure clusters
+|Deprecated
+|Deprecated
+|Deprecated
+
+|Package-based {op-system-base} compute machines
+|Deprecated
+|Deprecated
+|Deprecated
+
+|Managing machines with the Cluster API for {azure-full}
+|Not Available
+|Technology Preview
+|Technology Preview
```

Comment:
This somehow ended up in the wrong spot in 4.18. This goes in the Machine management TP table. I'll open a PR to fix 4.18.

Reply:
PR with 4.18 fix: https://github.com/openshift/openshift-docs/pull/89397

Reply:
:heavy_check_mark:

Reply:
thanks for merging that! :)